src/controllers/taskCommand.ts

```diff
@@ -0,0 +1,42 @@
+import { env } from "../typeDefinitions/default.types";
+import { TasksResponseType } from "../typeDefinitions/task.types";
+import { formatTask, generateTaskResponseMessage } from "../utils/formatTask";
+import { discordTextResponse } from "../utils/discordResponse";
+import { fetchTasks } from "../utils/fetchTasks";
+import { getNickName } from "../utils/getNickName";
+import {
+  INVALID_NICKNAME_ERROR,
+  NO_TASKS_FOUND,
+  TASKS_FETCH_FAILED,
+} from "../constants/responses";
+
+export async function taskCommand(userId: string, env: env) {
+  try {
+    const nickName = await getNickName(userId, env);
+    if (nickName === null) {
+      return discordTextResponse(INVALID_NICKNAME_ERROR);
+    }
+    const status = "IN_PROGRESS";
+    const tasksData = await fetchTasks(nickName, status);
+
+    if (!tasksData.tasks) {
+      const errorMessage = NO_TASKS_FOUND.replace("{{nickName}}", nickName);
```

Comment:
instead of replace we could have concatenated the nickname with the message `No tasks found for`

Reply:
Any issue with using this?

Reply:
concat will just concat the strings but replace will find the string and then replace so operation is increased, that's why I asked to do so